src/main/java/org/opensearch/security/configuration/ConfigurationRepository.java

```diff
@@ -461,7 +461,7 @@ public Map<CType, SecurityDynamicConfiguration<?>> getConfigurationsFromIndex(
             throw new OpenSearchException(e);
         }
 
-        if (logComplianceEvent && auditLog.getComplianceConfig().isEnabled()) {
+        if (logComplianceEvent && auditLog.getComplianceConfig() != null && auditLog.getComplianceConfig().isEnabled()) {
```

Comment:
Curious to know why this change was required?

Reply:
During the integration test when I was patching the security configuration to enable/disable OBO it errored out